src/utils/proposalFields.ts

```diff
@@ -1,27 +1,26 @@
-import { ApiBaseField, ApiProposal } from '../pdc-api';
+import type { Proposal, BaseField } from '@pdc/sdk';
 import { PROPOSAL_NAME_CASCADE, PROPOSAL_NAME_FALLBACK } from './proposals';
 
-const mapProposalBaseFields = (
-	baseFields: ApiBaseField[],
-	proposal: ApiProposal,
-) =>
+const mapProposalBaseFields = (baseFields: BaseField[], proposal: Proposal) =>
 	(proposal.versions[0]?.fieldValues ?? []).map(
 		({ applicationFormField, value }) => {
 			const baseField = baseFields.find(
+				// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
 				({ id }) => id === applicationFormField.baseFieldId,
 			);
 			return {
 				shortCode: baseField?.shortCode ?? 'missing',
 				fieldName: baseField?.label ?? 'missing',
+				// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment,@typescript-eslint/no-unsafe-member-access
```

Comment:
Reminder to add a comment explaining the reason for the eslint disable / linking to the opened issue (I won't tag all of them but you get the idea!)